File: C/iniciante/1010.c
#include <stdio.h>

typedef struct {
    int codigo;
    int num_peca;
    float preco;
} peca;

int main(){
    peca peca1, peca2;
    scanf("%d %d %f", &peca1.codigo, &peca1.num_peca, &peca1.preco);
    scanf("%d %d %f", &peca2.codigo, &peca2.num_peca, &peca2.preco);

    float total = peca1.num_peca*peca1.preco + peca2.num_peca * peca2.preco;
    printf("VALOR A PAGAR: R$ %.2f\n", total);

    return 0;
}

File: Python/Iniciante/1010.py
peca1 = str(input())
peca2 = str(input())
# Giving the variables
valoresPeca1 = peca1.split(" ")
valoresPeca2 = peca2.split(" ")
# Reutilizing him
valoresPeca1 = float(valoresPeca1[1])*float(valoresPeca1[2])
valoresPeca2 = float(valoresPeca2[1])*float(valoresPeca2[2])
# display the result
print("VALOR A PAGAR: "
      f"R$ {valoresPeca1 + valoresPeca2:.2f}")